libs/tx-bill/src/transitions/payBill.ts

```diff
@@ -1,11 +1,83 @@
-import { BillActionContainer, TransitionCallback } from "@thecointech/tx-statemachine";
+import { BillActionContainer, getCurrentState, TransitionCallback } from "@thecointech/tx-statemachine";
 import { verifyPreTransfer } from "@thecointech/tx-statemachine/transitions";
+import { BillPayeePacket, EncryptedPacket } from '@thecointech/types';
+import { readFileSync } from 'fs';
+import { decryptTo } from "@thecointech/utilities/Encrypt";
+import { sign } from "@thecointech/utilities/SignedMessages";
+import { log } from "@thecointech/logging";
+import { Decimal } from "decimal.js-light";
```

Comment:
nearly verbatim copy-pasted from eTransfer transition.  No tests added for this yet either